proto/storage/config.proto

```diff
@@ -43,6 +45,18 @@ message AlertRetentionConfig {
     int32 attempted_runtime_retention_duration_days = 5;
 }
 
+// next available tag: 4
+message DecommissionedClusterRetentionConfig {
+    // next available tag: 3
+    message IgnoreClusterLabel {
+        string key    = 1;
+        string value  = 2;
+    }
+    int32 retention_duration_days           = 1;
+    IgnoreClusterLabel ignore_label         = 2;
```

Comment:
nit: `ClusterLabel ignore_cluster_label`. Why are not multiple labels supported?

Reply:
+1, should be an array of labels. The UI will need to have some kind of delimiter in the textbox to separate multiple pairs. Assuming setting this label is available in the Add/Edit cluster workflow and the UI/UX on that is TBD like you alluded to in the standup today.

Reply:
It seems like a question for solution architects to answer how realistic the need, then for product to confirm a change in feature scope if needed.

Strictly from UI perspective, if it goes forward, for consistency with cluster labels, please imitate:
https://github.com/stackrox/stackrox/blob/master/proto/storage/cluster.proto#L100

Therefore, in response JSON:
* no label similar to current design:
    ```json
    "ignoreLabels": null
    ```
* one label:
    ```json
    "ignoreLabels": {"key1": "value1"}
    ```
* multiple labels:
    ```json
    "ignoreLabels": {"key1": "value1","key2":"value2"}
    ```

Reply:
Yeah thinking more about it, I think that makes sense to allow multiple labels in the system config rather than asking a user to add a new label to the cluster/s. I will make the change and push it.

Reply:
Done